### ISSUE TITLE
Emphasize importance of the training in adoption plan

### ISSUE THREAD

**Issue by jwillisrose** [PowerPlatformAF/PowerPlatformAF#48]
I see a brief mention of ABC in a Day training in the **Quick Start** and some mentions of training in the **Nurturing the Community** sections, but I think more emphasis could be made and perhaps explicit directions as to what types of skills are needed by whom (i.e. Citizen Developers should complete the .... learning path on Microsoft Learn).  There are many ways to acquire these skills and not everyone learns the same way, but it's critical to emphasize that everyone needs to prepare to learn as part of the adoption, whether they are a citizen developer or an IT admin.  

This may overlap with the CoE Starter Kit some, but again it's super important to build into any Power Platform adoption plan, this idea that the initial wave of employees you train will become your org's Power Platform subject matter experts (or champions) who deliver the training to the next generation of Power Platform users. Hopefully starting an endless cycle of training and learning that nurtures the community within your organization. 

In short, a good adoption plan creates a Power Platform training program that sustains the growth within the organization for years to come.
